storage/kvstore.go

```diff
@@ -489,3 +489,29 @@ func (s *store) delete(key []byte) {
 	}
 	s.currentRev.sub += 1
 }
+
+// appendMarkTombstone appends tombstone mark to the backend key.
```

Comment:
ok. revision without mark indicates a normal KV, and revision with mark indicates a tombstone kv. I will make the corresponded changes.